editor/editor_node.cpp

```diff
@@ -7117,6 +7114,8 @@ EditorNode::EditorNode() {
 	renderer->set_focus_mode(Control::FOCUS_NONE);
 	renderer->set_tooltip_text(TTR("Choose a rendering method.\n\nNotes:\n- On mobile platforms, the Mobile rendering method is used if Forward+ is selected here.\n- On the web platform, the Compatibility rendering method is always used."));
 
+	callable_mp(this, &EditorNode::_update_renderer_color).call_deferred();
```

Comment:
I've never seen this form of using `call_deferred()` in C++ before, but I guess it makes sense to avoid String usage. We should probably start using it in other places too.

Reply:
You you're already using it :)

https://github.com/godotengine/godot/blob/71699e08c9df78b7203fa4ef9cede28e995d6ace/editor/editor_node.cpp#L672

Here's the commit: https://github.com/godotengine/godot/commit/0e8f90f4c8b4b353d3ac372e5f00493a2f0bd136